google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -67,6 +101,11 @@ public static ReadAndQueryOption bufferRows(int bufferRows) {
     return new BufferRowsOption(bufferRows);
   }
 
+  /** Specifies the priority to use for the RPC. */
+  public static ReadQueryUpdateTransactionOption priority(RpcPriority priority) {
```

Comment:
Should we add the same helper method in the tagging feature? If so, could you add this feedback in the PR?

Reply:
I'm not sure whether you mean literally this helper method or a similar one for tagging. The latter is already in the PR:
https://github.com/googleapis/java-spanner/blob/c3be7ce852a8d88655fe295fda080143ff40123a/google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java#L81

But I don't think we should add this specific helper method to the other PR. The two PR's are going to conflict, but I think that is something we should fix once one of them has been merged.

Reply:
Oh sorry, I missed that from the other PR.